sample/Kmd.Logic.CitizenDocuments.Client.Sample/Program.cs

```diff
@@ -47,56 +46,59 @@ private static void InitLogger()
+                    ClientId = configuration.TokenProvider.ClientId,
+                    ClientSecret = configuration.TokenProvider.ClientSecret,
                 };
                 configuration.Citizen.SubscriptionId = configuration.SubscriptionId;
                 configuration.Citizen.Serviceuri = configuration.Serviceuri;
                 var tokenProviderFactory = new LogicTokenProviderFactory(tokenProviderOptions);
                 var citizenDocumentClient = new CitizenDocumentsClient(httpClient, tokenProviderFactory, configuration.Citizen);
-                var uploadDocument = await citizenDocumentClient.UploadAttachmentWithHttpMessagesAsync(configuration.ConfiguartionId, configuration.RetentionPeriodInDays, configuration.Cpr, configuration.DocumentType, configuration.Document, configuration.DocumentName).ConfigureAwait(false);
+                var uploadDocument = await citizenDocumentClient.UploadAttachmentWithHttpMessagesAsync(configuration.ConfigurationId, configuration.RetentionPeriodInDays, configuration.Cpr, configuration.DocumentType, configuration.Document, configuration.DocumentName).ConfigureAwait(false);
 
                 if (uploadDocument == null)
                 {
-                    Log.Error("There is error occured in upload");
-                    return;
+                    Log.Error("An unknown error occurred while uploading the document");
+                    return "An error occurred while uploading the citizen document";
                 }
-                Log.Information($"Document uploaded successfully and details are :-  DocumentId : {uploadDocument.DocumentId} ; DocumentType : {uploadDocument.DocumentType} ; FileAccessPageUrl : {uploadDocument.FileAccessPageUrl} ");
+
+                Log.Information("The {DocumentType} document with id {DocumentId} and file access page url {FileAccessPageUrl} is uploaded successfully", uploadDocument.DocumentType, uploadDocument.DocumentId, uploadDocument.FileAccessPageUrl);
 
                 var sendDocument = await citizenDocumentClient.SendDocumentWithHttpMessagesAsync(new SendCitizenDocumentRequest
                 {
-                    ConfigurationId = new Guid(configuration.ConfiguartionId),
+                    ConfigurationId = new Guid(configuration.ConfigurationId),
                     SendingSystem = configuration.SendingSystem,
                     Cpr = configuration.Cpr,
                     DocumentType = configuration.SendDocumentType,
                     CitizenDocumentId = uploadDocument.DocumentId,
-                    Title = configuration.title,
+                    Title = configuration.Title,
                     DigitalPostCoverLetterId = uploadDocument.DocumentId,
-                    SnailMailCoverLetterId = uploadDocument.DocumentId
-                });
+                    SnailMailCoverLetterId = uploadDocument.DocumentId,
+                }).ConfigureAwait(false);
+
                 if (sendDocument == null)
```

Comment:
Why do you explicitly handle `null` here? It looks like there's no reason to handle `null` anymore?

Reply:
Yeah correct now no reason to check null. Removed it.